app/src/main/java/cz/muni/fi/pv256/movio2/fk410022/drawable/GradientStar.java

```diff
@@ -0,0 +1,122 @@
+package cz.muni.fi.pv256.movio2.fk410022.drawable;
+
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.ColorFilter;
+import android.graphics.LinearGradient;
+import android.graphics.Paint;
+import android.graphics.Path;
+import android.graphics.PixelFormat;
+import android.graphics.Rect;
+import android.graphics.Shader;
+import android.graphics.drawable.Drawable;
+
+/**
+ * inspired by http://www.therealjoshua.com/2012/12/imageviews-and-stars/
+ */
+public class GradientStar extends Drawable {
```

Comment:
ten layout s textom a hodnotenim by mal byt transparentny s gradientom, nie zrovna hviezdicka :)))  hm, mozno je to chyba v zadani....